OutOfSchool/OutOfSchool.RazorTemplatesData/Views/Emails/NewAdminInvitation/NewAdminInvitation.cshtml

```diff
@@ -1,5 +1,25 @@
-﻿@*
-    For more information on enabling MVC for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
-*@
+﻿@using OutOfSchool.RazorTemplatesData.Models.Emails
+@using OutOfSchool.RazorTemplatesData.Models.Shared
+@model AdminInvitationViewModel
+
 @{
+    ViewData["EmailTitle"] = "Welcome!";
 }
+
+<p>Для реєстрації на платформі перейдіть за посиланням та заповність ваші данні в особистому кабінеті.</p>
+
+<br />
+
+<p>Логін: @Model.Email</p>
+
+<br />
+
+<p>Пароль:  @Model.Password</p>
+
+<br />
+
+@await Html.PartialAsync("EmailButton", new EmailButtonViewModel("Confirm Email", Model.ConfirmationUrl))
+
+<br />
+
+<p>If you have any questions, just reply to this email—we're always happy to help out.</p>
```

Comment:
Please translate last paragraph to Ukrainian or remove it

Reply:
Ok, but please don't increase entropy by adding mistakes from the very beginning. 
I'm bit afraid of situation that mentioned frontend team might blindly reuse you texts and thus error will slip into final version of code.